src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -308,6 +308,7 @@ public static class Release
                 // Keep alphabetical
                 //
                 public static readonly string AgentReleaseDirectory = "agent.releaseDirectory";
+                public static readonly string AgentReleaseRootDirectory = "agent.releaseRootDirectory";
```

Comment:
interesting. sounds like a bug in the temp directory cleanup not waiting for the upload queue to drain